Assets/Hai/DestroyEditorOnly/DestroyEditorOnly.cs

```diff
@@ -57,6 +57,13 @@ private static void TryDestroyRecursive(GameObject subject, bool destroyPermitte
     [MenuItem("Tools/Enable DestroyEditorOnly")]
     public static void EnableDestroyEditorOnly()
     {
+        var existComponent = FindObjectOfType<DestroyEditorOnly>();
+        if (existComponent != null)
+        {
+            existComponent.gameObject.SetActive(true);
```

Comment:
`FindObjectOfType` will not return GameObjects that are disabled (themselves or any of its parents), so when this line is entered, the `existComponent.gameObject` is already guaranteed to be active.

It may be necessary to use alternatives like `<???>.GetComponentInChildren<DestroyEditorOnly>(true)`